src/renderer/components/Wallet/WalletHeading/WalletHeadingPrimaryActions.vue

```diff
@@ -111,17 +110,10 @@ export default {
 
     doesNotExist () {
       return !this.$store.getters['wallet/byAddress'](this.currentWallet.address)
-    }
-  },
-
-  watch: {
-    // Never show the not-voting icon until knowing if the wallet is voting or not
-    'currentWallet.address' () {
-      this.isVoting = false
     },
-    // To react to changes on the injected `walletVote` and changed not-voting icon immediately
-    'walletVote.username' () {
-      this.isVoting = !!this.walletVote.username
+
+    isVoting () {
+      return !!this.walletVote.username
```

Comment:
Hm, `walletVote` is not being updated when navigating between wallets, need to click the refresh button. It must be cleaned and updated when the `currentWallet` changes.

Reply:
I'll check/fix in a bit, thanks for pointing it out!

Reply:
I'm unable to reproduce. Switching between a voting and a non-voting wallet also changes the dot as intended. Am I missing something?

Reply:
Try browsing through non-imported wallets. Delegates tab > Open any delegate > Click on the address.

Reply:
Gotcha, i'll have a look ASAP.

Reply:
It worked when switching through the sidebar since that caused `fetchWalletVote` to be called. I changed it to call that method when `currentWallet` changes instead.